test/session-client.test.ts

```diff
@@ -23,15 +23,16 @@ class TestSRPClientSession extends SRPClientSession {
     if (startingState) {
       this.stateStep = startingState;
 
-      switch (this.stateStep) {
-        case SRPClientSessionState.STEP_3:
-        case SRPClientSessionState.STEP_2:
-          this.assumeStep2();
```

Comment:
Look like a `break` was missing here

Reply:
Adding one broke a test. Apparently this code was running both assumeStep2 and assumeStep1 if stateStep was STEP_3 OR STEP_2. Quite confusing.